notebooks/working/external_notebook_skeleton.ipynb

```diff
@@ -0,0 +1,219 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Notebook summary\n",
+    "\n",
+    "A paragraph about what we're showing in this notebook, potentially with a table of content that links to each subsection of the notebook. People should be able to read this and understand the flow as well as the main take-away from this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Import packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Set work directory and load the dataset. "
```

Comment:
Just for good practice, can you designate an earlier cell to store notebook parameters / global constants? Things like filepaths to data, the number of simulations to be executed, random seeds, filepaths to store the plots created at the end, etc.